src/app/missing_subchain/block.ml

```diff
@@ -233,24 +246,40 @@ let internal_commands_of_extensional_block internal_cmds_tbl
   in
   List.map internal_cmds ~f:internal_cmd_balance_data_of_internal_cmd
 
-let diff_commands_of_extensional_block user_cmds_tbl internal_cmds_tbl block =
-  { commands= user_commands_of_extensional_block user_cmds_tbl block
-  ; internal_command_balances=
-      List.filter_opt
-        (internal_commands_of_extensional_block internal_cmds_tbl block) }
+let diff_commands_of_extensional_block user_cmds_tbl internal_cmds_tbl block :
+    Staged_ledger_diff.Diff.t =
+  let prediff_with_at_most_two_coinbase =
+    { Staged_ledger_diff.Pre_diff_two.completed_works= [] (* DUMMY *)
+    ; commands= user_commands_of_extensional_block user_cmds_tbl block
+    ; coinbase= Zero (* DUMMY *)
+    ; internal_command_balances=
+        List.filter_opt
+          (internal_commands_of_extensional_block internal_cmds_tbl block) }
+    (* REVIEWER: does it make sense to put all the commands in the at-most-two part? *)
```

Comment:
I think the block needs to be faithful to the original layout, so that the sequence numbers are correct. Otherwise, all of the fee transfers (from `completed_works` above) will end up at the end.